client/src/test/testUtils.js

```diff
@@ -0,0 +1,12 @@
+import thunk from "redux-thunk";
+import configureStore from "redux-mock-store";
+
+const mockStore = configureStore([thunk]);
+
+export const makeMockStore = (state = {}) => {
+  return mockStore({ ...state });
+};
```

Comment:
Seems like an unnecessary function. Why not just export `mockStore` and call it directly instead?